packages/react/src/ComparisonTable/ComparisonTable.stories.tsx

```diff
@@ -0,0 +1,169 @@
+import React from 'react'
+import {ComponentMeta, ComponentStory} from '@storybook/react'
+import {INITIAL_VIEWPORTS} from '@storybook/addon-viewport'
+
+import {Text, InlineLink} from '../'
+import {ComparisonTable} from '.'
+
+export default {
```

Comment:
The default values don't seem to be showing in Storybook for whatever reason. I think once those are added the docs should be good for this component. I added the defaults to the args object in this main default rather than by the individual argType.

Reply:
Do you mean the Storybook Docs? I haven't checked those, or use those regularly. Marketing Eng and customers typically use our Doctocat docs too, so I don't think we need to worry about this right now.

Reply:
I think this is part of the ADR that Katie was creating to always include these.

Reply:
These defaults are working for me, but sometimes they are finicky. I think this code looks good 👍